ThesisTemplate.tex

```diff
@@ -381,29 +377,29 @@ \section{Nuclear Fuel Cycle Simulators}
 \begin{itemize}
 \item "Characterize and deploy individual fuel cycle facilities and reactors in a simulation while discretely tracking material movements,
 \item The capability to perform an uncertainty analysis for each element of the fuel cycle and an aggregate uncertainty analysis,
-\item Inclusion of an optimization engine able to optimize simultaneously across multiple objective functions, and 
-\item Open and accessible code software and documentation to aid in collaboration between multiple entities and to facilitate software updates." 
+\item Inclusion of an optimization engine able to optimize simultaneously across multiple objective functions, and
```

Comment:
Optimization to optimize? You don't say.

Reply:
This is a quote, so I am not going to change it.